core/api/src/services/ledger/schema.ts

```diff
@@ -40,6 +42,11 @@ const transactionSchema = new Schema<ILedgerTransaction>(
       required: true,
     },
 
+    state: {
```

Comment:
I changed it ([`force-push`](https://github.com/GaloyMoney/galoy/compare/ed65bba06dc90603d89089827f845530e04615f1..7facaa14813161609dbfc6c1bba6f4f262c11c9c)) to `bundle_completion_state` to be more specific. It should be generic in case we want to start tagging more bundle types. For now we're just tagging ln-payment-related bundles